WordPress/gutenberg
# Use ClipboardJS latest version and clean up focus loss workaround.

Fixes https://github.com/WordPress/gutenberg/issues/57154

## What?

Upgrades ClipboardJS to 2.0.11.
Removes a focus loss workaround.

## Why?

ClipboardJS 2.0.11 includes a fix for a focus loss that makes our workaround no longer necessary.
WordPress Core already uses 2.0.11.

## How?

Upgrades ClipboardJS.
Removes focus loss fix from `useCopyToClipboard`.

## Testing Instructions




- Test the original issue where we introduced our foucs loss workaround https://github.com/WordPress/gutenberg/issues/23994
- Select a paragraph block.
- Use the keyboard to move focus to the block toolbar, open the 'more' Options menu and activate the 'Copy' button with the Enter or Spacebar key.
- Observe the dropdown menu stays open.
- Observe focus stays on the 'Copy' button.
- Press the Tab key and observe focus moves to the next item in the dropdown menu.
- Paste your clipboard content into a text editor and observe the block content has been copied correctly.
- Open the Color picker by clicking 'Text' in the settings panel.
- Click the color preview to set a custom color. The custom color picker opens.
- Set a color.
- Click the Copy icon.
- Observe the color picker stays open and focus stays on the Copy button.
- Paste your clipboard content into a text editor and observe the color code has been copied correctly.


### Testing Instructions for Keyboard


## Screenshots or screencast
